addOns/reports/src/test/java/org/zaproxy/addon/reports/ExtensionReportsUnitTest.java

```diff
@@ -1097,12 +1120,11 @@ void shouldGenerateValidXmlReport() throws Exception {
         dbf.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
         DocumentBuilder db = dbf.newDocumentBuilder();
 
-        // When
+        // Then
```

Comment:
This should be kept `When`, it's the other statements that need to be moved to under `Then`.